stargate/crates/query-planner/src/builder.rs

```diff
@@ -364,11 +364,11 @@ fn complete_field<'a, 'q: 'a>(
         // the type_name could be a primitive type which is not in our names_to_types map.
         let return_type = context.names_to_types.get(type_name);
 
-        if return_type.is_none() || !return_type.unwrap().is_composite_type() {
+        if return_type.is_none() || !return_type.expect("checked not None").is_composite_type() {
```

Comment:
i think this would be better as an `if let`

Reply:
How? afaik I can't add a condition with an `||` to an `if let`